### ISSUE TITLE
ci{ does not work

### ISSUE THREAD

**Issue by gkabbe** [t9md/atom-vim-mode-plus#785]
# Check list

atom --version:

```
Atom    : 1.16.0
Electron: 1.3.13
Chrome  : 52.0.2743.82
Node    : 6.5.0
```

vim-mode-plus version: 0.92.1

OS: Debian Jessie

vim-mode is not installed

Fresh atom installation, and latest vim-mode-plus (0.92.1)

While editing Latex documents, I noticed that ci{ does not work.

Example document with single line:
```
{Test bracket}
```

Cursor is on first letter of  word "bracket"

Input: 
```
ci{
```

Desired result:
```
{}
```

Actual result:
```
bracket}
```
According to the Keybinding-Resolver, the bracket in the command ci{ is interpreted as the command move-to-previous-paragraph.

Similarly, ci[ does not work, as the "[" is interpreted as the command move-up-to-edge

**Comment by t9md** [comment]
From keybinding resokver's info, when you typed ci it's not correctly entering operator-pending-mode.
Is this happen in other file types?
Removing your custom keymap solve your issue?
How about di{? How about cw?

**Comment by gkabbe** [comment]
* I tried out some different file types (.py, .tex, .c, .cpp), all with the same result.
* My custom keymap is empty, but it still occurs.
* di{ for "{Test bracket}" also results in "bracket}", whereas di[ applied to "[Test bracket]" results in "[Test bracket]" (i.e. nothing is deleted at all).
* cw works perfectly fine (i.e. results in "{Test }")

**Comment by t9md** [comment]
How about `c i p`?, `c i w`.
I need this to know text-object target is working in your environment.

In case you didn't know, atom's keybinding system have 1sec timeout for partial-match keybinding.
So when you type `c i` then your next `p` or `{` or `[` is too slow, it treated as just `{` instead of not as `i {`.
The behavior you reported seem to match this behavior.
How about your keystroke speed when issue happening?

**Comment by gkabbe** [comment]
Sorry for the late answer..
```cip``` and ```ciw``` both work fine.

For example,

```
abc
def

ghi
```
if the cursor is on d, and I type ```ciw```, I get

```
abc


ghi
```

With ```cip``` I get

```


ghi
```

I think the keystroke speed is not the issue (I typed extra fast this time ;) ).

But if I am the only one with this issue, I will check if there is not something else wrong with my setup.

**Comment by dylnmc** [comment]
`ci{`, `ci}`, `di{`, `di}`, `vi{`, `vi}`, `ca}`, `ca}`, `da{`, `da}`, `va{`, `va}`  all work for me ... in multi-line and single-line curly brackets.

atom version: 1.17.2 (x64)
vim-mode-plus version: 0.93.0

**Comment by turnspike** [comment]
I'm having the same issue as @gkabbe 
Platform: Macos 10.12.6
Atom: 1.22.0
vim-mode-plus: 1.16.0

`ci'` and `ci"` work great
`ci{` `ci(` and `ci[` don't work at all

I'm guessing something is blatting the keymaps but I don't know enough about Atom to debug further.

**Comment by t9md** [comment]
I believe this issue can be resolved by reading #668, #661.
Close in favor of #668, #661.
